hack/test-pre-pull-images.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+echo "[INFO] Prepulling container images"
+
+# https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest
+# This list of images was created via:
+# grep -Iiroh "gcr.io/google_.*" "${OS_ROOT}" | \
+#     sed -e "s/[,\")}]//g" | awk '{print $1}' | sort | uniq | tr '\n' ' '
+
+for i in gcr.io/google_containers/busybox \
```

Comment:
@liggitt that makes sense. Not sure about clean up just picked it up from that kube's manifest. 
@ncdc will update soon.